tutorial/tutorial_tensor_core.ipynb

```diff
@@ -0,0 +1,1110 @@
+    "                T.float32(0),\n",
+    "                dtype=\"handle\",\n",
+    "            )\n",
+    "        )\n",
+    "\n",
+    "\n",
+    "@T.prim_func\n",
+    "def wmma_store_desc(a: T.handle, c: T.handle) -> None:\n",
+    "    A = T.match_buffer(\n",
+    "        a, (16, 16), \"float32\", align=128, offset_factor=16, scope=\"wmma.accumulator\"\n",
+    "    )\n",
+    "    C = T.match_buffer(c, (16, 16), \"float32\", align=128, offset_factor=16, scope=\"global\")\n",
+    "    with T.block(\"root\"):\n",
+    "        T.reads(A[0:16, 0:16])\n",
+    "        T.writes(C[0:16, 0:16])\n",
+    "        for i, j in T.grid(16, 16):\n",
+    "            with T.block(\"store\"):\n",
+    "                vii, vjj = T.axis.remap(\"SS\", [i, j])\n",
+    "                C[vii, vjj] = A[vii, vjj]\n",
+    "\n",
+    "\n",
+    "@T.prim_func\n",
+    "def wmma_store_impl(a: T.handle, c: T.handle) -> None:\n",
+    "    s1 = T.var(\"int32\")\n",
+    "    s0 = T.var(\"int32\")\n",
+    "    A = T.match_buffer(\n",
+    "        a, (16, 16), \"float32\", align=128, offset_factor=16, scope=\"wmma.accumulator\"\n",
+    "    )\n",
+    "    C = T.match_buffer(\n",
+    "        c,\n",
+    "        (16, 16),\n",
+    "        \"float32\",\n",
+    "        align=128,\n",
+    "        offset_factor=16,\n",
+    "        scope=\"global\",\n",
+    "        strides=[s1, s0],\n",
+    "    )\n",
+    "    with T.block(\"root\"):\n",
+    "        T.reads(A[0:16, 0:16])\n",
+    "        T.writes(C[0:16, 0:16])\n",
+    "        T.evaluate(\n",
+    "            T.tvm_store_matrix_sync(\n",
+    "                A.data,\n",
+    "                16,\n",
+    "                16,\n",
+    "                16,\n",
+    "                A.elem_offset // 256 + T.floordiv(T.floormod(A.elem_offset, 256), 16),\n",
+    "                C.access_ptr(\"w\"),\n",
+    "                s1,\n",
+    "                \"row_major\",\n",
+    "                dtype=\"handle\",\n",
+    "            )\n",
+    "        )\n",
+    "\n",
+    "\n",
+    "try:\n",
+    "    # handle exception if we register multi times\n",
+    "    tir.TensorIntrin.register(\"wmma_load_a\", wmma_load_a_desc, wmma_load_a_impl)\n",
+    "    tir.TensorIntrin.register(\"wmma_load_b\", wmma_load_b_desc, wmma_load_b_impl)\n",
+    "    tir.TensorIntrin.register(\"wmma_sync\", wmma_sync_desc, wmma_sync_impl)\n",
+    "    tir.TensorIntrin.register(\"wmma_fill\", wmma_fill_desc, wmma_fill_impl)\n",
+    "    tir.TensorIntrin.register(\"wmma_store\", wmma_store_desc, wmma_store_impl)\n",
+    "except ValueError:\n",
+    "    pass"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Blockize the Tensorized Computation\n",
+    "As says in the lecture, we can use TensorIR to represent a group of tensorized computation with a `Block`. We can directly write a TensorIR program with `Block`, but also can tile the loops and `blockize` it."
```

Comment:
Before this section, we haven't mentioned that wmma intrin does 16x16x16 computation. We need to explain it so that students can know why the split factors are 16.